draft-ietf-quic-transport.md

```diff
@@ -73,9 +73,11 @@ informative:
     title: "Structured Streams: A New Transport Abstraction"
     author:
       - ins: B. Ford
-    date: 2007-08
+    date: 2007-10
```

Comment:
That would give you: https://xml2rfc.tools.ietf.org/public/rfc/bibxml-doi/reference.DOI.10.1145/1282427.1282421.xml?anchor=SST

~~~
<reference anchor='SST' >
  <front>
    <title>Structured streams</title>
    <author initials="B." surname="Ford" fullname="Bryan Ford">
      <organization></organization>
    </author>
    <date year="2007" month="October"/>
  </front>
  <seriesInfo name="ACM SIGCOMM Computer Communication Review" value="Vol. 37, pp. 361"/>
  <seriesInfo name="DOI" value="10.1145/1282427.1282421"/>
</reference>
~~~

...which is cool, but, for instance, doesn't have the direct link to the actual paper.